cache/redisOps.js

```diff
@@ -46,6 +47,23 @@ function hmSet(objectName, name, value) {
 } // hmSet
 
 /**
+ * Get the value that is mapped to a key
+ * @param  {String} type - The type of the object on which the operation is to
+ * be performed
+ * @param  {String} name -  Name of the key
+ * @returns {Promise} - which resolves to the value associated with the key
+ */
+function getValue(type, name) {
```

Comment:
There is already a function getHashPromise with same intent. This method converts arrayStringsToJson as well. Any way we can avoid code duplication? If you insist on keeping the function, we should atleast change the name and comments to reflect how this function is different from other.